src/makeCalculator.js

```diff
@@ -37,7 +37,40 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calcuate = {
+    result: 0,
+    operate(callback, value) {
+      callback(value);
+
+      return calcuate;
```

Comment:
```suggestion
      return this;
```
не нужно хардкодить название объекта. используй this